### ISSUE TITLE
Not compatible with Brackets-AngularJS-CodeHints

### ISSUE THREAD

**Issue by ghost** [chrisgriffith/IonicBrackets#6]
It stopped working when I installed: https://github.com/sirajc/Brackets-AngularJS-CodeHints/

**Comment by victorfeitosa** [comment]
This is actually not compatible with any codehint that overlaps it (like any HTML code hint plugin such as Angular, Polymer, etc). Although it already has angular properties on its own, it's a annoying task having to disable this plugin whenever working with something else and turning it back on when comming back to Ionic.

**Comment by chrisgriffith** [comment]
That is correct. It seems that is how the Bracket's code hinter works ( which this is based on).
